<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Elisa Rubio" />

<meta name="date" content="2022-08-21" />

<title>06_biplot</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">2022_Rubio_MetaHIV</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="01_import_data.html">Import data</a>
</li>
<li>
  <a href="02_summarize_data.html">Summarize data</a>
</li>
<li>
  <a href="03_rarefaction.html">Rarefy data</a>
</li>
<li>
  <a href="04_alpha_div.html">Alpha diversity analysis</a>
</li>
<li>
  <a href="05_beta_div.html">Beta diversity analysis</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">06_biplot</h1>
<h4 class="author">Elisa Rubio</h4>
<h4 class="date">2022-08-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-08-23
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>2022_Rubio_MetaHIV/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220202code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220202)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220202code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220202)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomMicrobialGenomics2022RubioMetaHIVtree67563617bad643376f87ef8b2286f5a2ae7d9c15targetblank6756361a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/MicrobialGenomics/2022_Rubio_MetaHIV/tree/67563617bad643376f87ef8b2286f5a2ae7d9c15" target="_blank">6756361</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomMicrobialGenomics2022RubioMetaHIVtree67563617bad643376f87ef8b2286f5a2ae7d9c15targetblank6756361a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/MicrobialGenomics/2022_Rubio_MetaHIV/tree/67563617bad643376f87ef8b2286f5a2ae7d9c15" target="_blank">6756361</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    output/aribaData/
    Ignored:    output/grootData/

Unstaged changes:
    Modified:   analysis/06_biplot.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/06_biplot.Rmd</code>) and HTML
(<code>docs/06_biplot.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/MicrobialGenomics/2022_Rubio_MetaHIV/blob/9020ce61bccd5c1f70226ffa49b992ee13f93423/analysis/06_biplot.Rmd" target="_blank">9020ce6</a>
</td>
<td>
Elisa
</td>
<td>
2022-08-22
</td>
<td>
biplot 220822
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/MicrobialGenomics/2022_Rubio_MetaHIV/blob/c5f0d18f50cf2ffa00576cb05104b03c0271f2ef/analysis/06_biplot.Rmd" target="_blank">c5f0d18</a>
</td>
<td>
Elisa
</td>
<td>
2022-08-21
</td>
<td>
biplot test
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(tidyverse)
library(vegan)
library(broom)
library(ggrepel)
library(glue)
library(usedist)</code></pre>
<pre class="r"><code>load(&quot;output/ariba_dist.RDA&quot;)
load(&quot;output/groot_dist.RDA&quot;)
load(&quot;output/summarized_ariba.RDA&quot;)
load(&quot;output/summarized_groot.RDA&quot;)
load(&quot;output/rarefied_ariba.RDA&quot;)
load(&quot;output/rarefied2500_groot.RDA&quot;)
metadata&lt;-read_csv(&quot;data/Metadata/metadata.csv&quot;)</code></pre>
<p>Correlations of AMR genes with NMDs axes:</p>
<pre class="r"><code>biplot_amr&lt;-function(data, metadata, clin_var, cor_level){

clin_var&lt;-enquo(clin_var)  
dist&lt;-vegdist(t(data[ ,-1]), method = &quot;bray&quot;); dist&lt;- dist_setNames(dist, colnames(data)[-1])
set.seed(200889)
nmds &lt;- metaMDS(dist)
##nmds_positions&lt;-scores(nmds)%&gt;%as_tibble(rownames=&quot;SampleID&quot;)

nmds_positions&lt;- nmds$points %&gt;%
  as_tibble(rownames=&quot;SampleID&quot;)%&gt;%
  rename(NMDS1=MDS1, NMDS2=MDS2)

ref_name&lt;-pull(data, ref_name)
data&lt;-as_tibble(cbind(SampleID = names(data), t(data)))%&gt;%slice(-1) 
colnames(data)&lt;-c(&quot;SampleID&quot;, ref_name)
data&lt;-data%&gt;%pivot_longer(-SampleID)

nmds_shared &lt;- inner_join(data, nmds_positions)%&gt;%mutate(value=as.numeric(value))

cor_x &lt;- nmds_shared %&gt;%
  nest(data = -name) %&gt;%
  mutate(cor_x = map(data,
                     ~cor.test(.x$value, .x$NMDS1,
                               method=&quot;spearman&quot;,
                               exact=FALSE) %&gt;% tidy())) %&gt;%
  unnest(cor_x) %&gt;%
  select(name, estimate, p.value)
  
cor_y &lt;- nmds_shared %&gt;%
  nest(data = -name) %&gt;%
  mutate(cor_y = map(data,
                     ~cor.test(.x$value, .x$NMDS2,
                               method=&quot;spearman&quot;,
                               exact=FALSE) %&gt;% tidy())) %&gt;%
  unnest(cor_y) %&gt;%
  select(name, estimate, p.value)

correlations &lt;- inner_join(cor_x, cor_y, by=&quot;name&quot;)

high_corr &lt;- correlations %&gt;%
  filter(estimate.x&lt;=0.05 | estimate.y&lt;=0.05)%&gt;%
  filter(abs(estimate.x) &gt; cor_level | abs(estimate.y) &gt; cor_level) 

nmds_positions%&gt;%
  left_join(., metadata, by=&quot;SampleID&quot;) %&gt;%
  ggplot(aes(x=NMDS1, y=NMDS2, color=!!clin_var)) +
  geom_point(alpha=0.5, size=2)+
  geom_segment(data=high_corr,
               aes(x=0, xend=estimate.x, y=0, yend=estimate.y), 
               arrow = arrow(length = unit(0.2, &quot;cm&quot;)), alpha=1,colour=&quot;gray&quot;,
               inherit.aes=FALSE)+
  geom_text_repel(data=high_corr,
                  aes(x=estimate.x, y=estimate.y, label=name),
                  min.segment.length = 0.15, segment.alpha=1, segment.color=&quot;gray&quot;,
                  inherit.aes=FALSE) +
  theme_bw()+
  scale_color_manual(values = c(&quot;chartreuse4&quot;, &quot;red&quot;, &quot;gray&quot;))+
  labs(caption = glue(&quot;Gene level, {cor_level} spearman correlation&quot;))}</code></pre>
<p>Correlations of numerical metadata variables with NMDS axes</p>
<pre class="r"><code>biplot_meta&lt;-function(data, metadata, clin_var, cor_level){

clin_var&lt;-enquo(clin_var)  

dist&lt;-vegdist(t(data[ ,-1]), method = &quot;bray&quot;); dist&lt;- dist_setNames(dist, colnames(data)[-1])
set.seed(200889)
nmds &lt;- metaMDS(dist)
##nmds_positions&lt;-scores(nmds)%&gt;%as_tibble(rownames=&quot;SampleID&quot;)

nmds_positions&lt;- nmds$points %&gt;%
  as_tibble(rownames=&quot;SampleID&quot;)%&gt;%
  rename(NMDS1=MDS1, NMDS2=MDS2)


meta&lt;-metadata%&gt;%select(SampleID, where(is.numeric))%&gt;%
  pivot_longer(-SampleID, values_drop_na = TRUE)

nmds_shared &lt;- inner_join(nmds_positions, meta)%&gt;%mutate(value=as.numeric(value))

cor_x &lt;- nmds_shared %&gt;%
  nest(data = -name) %&gt;%
  mutate(cor_x = map(data,
                     ~cor.test(.x$value, .x$NMDS1,
                               method=&quot;spearman&quot;,
                               exact=FALSE) %&gt;% tidy())) %&gt;%
  unnest(cor_x) %&gt;%
  select(name, estimate, p.value)
  
cor_y &lt;- nmds_shared %&gt;%
  nest(data = -name) %&gt;%
  mutate(cor_y = map(data,
                     ~cor.test(.x$value, .x$NMDS2,
                               method=&quot;spearman&quot;,
                               exact=FALSE) %&gt;% tidy())) %&gt;%
  unnest(cor_y) %&gt;%
  select(name, estimate, p.value)

correlations &lt;- inner_join(cor_x, cor_y, by=&quot;name&quot;)

high_corr &lt;- correlations %&gt;%
  filter(estimate.x&lt;=0.01 | estimate.y&lt;=0.01)%&gt;%
  filter(abs(estimate.x) &gt; cor_level | abs(estimate.y) &gt; cor_level) 

if (nrow(high_corr)==0){stop(&quot;No significant correlations&quot;)
  } else {
nmds_positions%&gt;%
  left_join(., metadata, by=&quot;SampleID&quot;) %&gt;%
  ggplot(aes(x=NMDS1, y=NMDS2, color=!!clin_var)) +
  geom_point(alpha=0.5, size=2)+
  geom_segment(data=high_corr,
               aes(x=0, xend=estimate.x, y=0, yend=estimate.y), 
               arrow = arrow(length = unit(0.2, &quot;cm&quot;)), alpha=1,colour=&quot;gray&quot;,
               inherit.aes=FALSE)+
  geom_text_repel(data=high_corr,
                  aes(x=estimate.x, y=estimate.y, label=name),
                  min.segment.length = 0.15, segment.alpha=1, segment.color=&quot;gray&quot;,
                  inherit.aes=FALSE) +
  theme_bw()+
  scale_color_manual(values = c(&quot;chartreuse4&quot;, &quot;red&quot;, &quot;gray&quot;))+
  labs(caption = glue(&quot;{cor_level} spearman correlation&quot;))}
}</code></pre>
<div id="ariba" class="section level2">
<h2>ARIBA</h2>
<pre class="r"><code>biplot_amr(ariba_rar, metadata, RiskGroup2, 0.5)+labs(title = &quot;Ariba Rarefied counts&quot;)</code></pre>
<pre><code>Run 0 stress 0.2619139 
Run 1 stress 0.2630689 
Run 2 stress 0.2623743 
... Procrustes: rmse 0.01727915  max resid 0.109121 
Run 3 stress 0.2617397 
... New best solution
... Procrustes: rmse 0.02405941  max resid 0.2271366 
Run 4 stress 0.2658973 
Run 5 stress 0.2660067 
Run 6 stress 0.2619111 
... Procrustes: rmse 0.0172414  max resid 0.2050288 
Run 7 stress 0.2666935 
Run 8 stress 0.2637621 
Run 9 stress 0.2715775 
Run 10 stress 0.2637698 
Run 11 stress 0.2670776 
Run 12 stress 0.2640943 
Run 13 stress 0.2634628 
Run 14 stress 0.2665091 
Run 15 stress 0.2660558 
Run 16 stress 0.2630005 
Run 17 stress 0.2655312 
Run 18 stress 0.2642922 
Run 19 stress 0.2641465 
Run 20 stress 0.2687306 
*** No convergence -- monoMDS stopping criteria:
     4: no. of iterations &gt;= maxit
    16: stress ratio &gt; sratmax</code></pre>
<p><img src="figure/06_biplot.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>biplot_meta(ariba_rar, metadata, RiskGroup2, 0.2)+labs(title = &quot;Ariba Rarefied counts&quot;)</code></pre>
<pre><code>Run 0 stress 0.2619139 
Run 1 stress 0.2630689 
Run 2 stress 0.2623743 
... Procrustes: rmse 0.01727915  max resid 0.109121 
Run 3 stress 0.2617397 
... New best solution
... Procrustes: rmse 0.02405941  max resid 0.2271366 
Run 4 stress 0.2658973 
Run 5 stress 0.2660067 
Run 6 stress 0.2619111 
... Procrustes: rmse 0.0172414  max resid 0.2050288 
Run 7 stress 0.2666935 
Run 8 stress 0.2637621 
Run 9 stress 0.2715775 
Run 10 stress 0.2637698 
Run 11 stress 0.2670776 
Run 12 stress 0.2640943 
Run 13 stress 0.2634628 
Run 14 stress 0.2665091 
Run 15 stress 0.2660558 
Run 16 stress 0.2630005 
Run 17 stress 0.2655312 
Run 18 stress 0.2642922 
Run 19 stress 0.2641465 
Run 20 stress 0.2687306 
*** No convergence -- monoMDS stopping criteria:
     4: no. of iterations &gt;= maxit
    16: stress ratio &gt; sratmax</code></pre>
<p><img src="figure/06_biplot.Rmd/unnamed-chunk-5-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>biplot_amr(ariba_rpkm, metadata, RiskGroup2, 0.5)+labs(title = &quot;Ariba RPKM&quot;)</code></pre>
<pre><code>Run 0 stress 0.2486242 
Run 1 stress 0.2486689 
... Procrustes: rmse 0.002925312  max resid 0.03230969 
Run 2 stress 0.2496518 
Run 3 stress 0.2486503 
... Procrustes: rmse 0.002718464  max resid 0.03223658 
Run 4 stress 0.2488234 
... Procrustes: rmse 0.007291962  max resid 0.07156253 
Run 5 stress 0.2487869 
... Procrustes: rmse 0.006795064  max resid 0.07183843 
Run 6 stress 0.2524771 
Run 7 stress 0.2488573 
... Procrustes: rmse 0.006485758  max resid 0.07181188 
Run 8 stress 0.249641 
Run 9 stress 0.2498946 
Run 10 stress 0.2488563 
... Procrustes: rmse 0.007030834  max resid 0.07164504 
Run 11 stress 0.2496531 
Run 12 stress 0.2486444 
... Procrustes: rmse 0.00108442  max resid 0.01049109 
Run 13 stress 0.2486166 
... New best solution
... Procrustes: rmse 0.001826592  max resid 0.01580941 
Run 14 stress 0.2516208 
Run 15 stress 0.2486463 
... Procrustes: rmse 0.002707593  max resid 0.03221828 
Run 16 stress 0.2495862 
Run 17 stress 0.2496288 
Run 18 stress 0.2496519 
Run 19 stress 0.2496403 
Run 20 stress 0.2496276 
*** No convergence -- monoMDS stopping criteria:
     1: no. of iterations &gt;= maxit
    19: stress ratio &gt; sratmax</code></pre>
<p><img src="figure/06_biplot.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>biplot_meta(ariba_rpkm, metadata, RiskGroup2, 0.2)+labs(title = &quot;Ariba RPKM&quot;)</code></pre>
<pre><code>Run 0 stress 0.2486242 
Run 1 stress 0.2486689 
... Procrustes: rmse 0.002925312  max resid 0.03230969 
Run 2 stress 0.2496518 
Run 3 stress 0.2486503 
... Procrustes: rmse 0.002718464  max resid 0.03223658 
Run 4 stress 0.2488234 
... Procrustes: rmse 0.007291962  max resid 0.07156253 
Run 5 stress 0.2487869 
... Procrustes: rmse 0.006795064  max resid 0.07183843 
Run 6 stress 0.2524771 
Run 7 stress 0.2488573 
... Procrustes: rmse 0.006485758  max resid 0.07181188 
Run 8 stress 0.249641 
Run 9 stress 0.2498946 
Run 10 stress 0.2488563 
... Procrustes: rmse 0.007030834  max resid 0.07164504 
Run 11 stress 0.2496531 
Run 12 stress 0.2486444 
... Procrustes: rmse 0.00108442  max resid 0.01049109 
Run 13 stress 0.2486166 
... New best solution
... Procrustes: rmse 0.001826592  max resid 0.01580941 
Run 14 stress 0.2516208 
Run 15 stress 0.2486463 
... Procrustes: rmse 0.002707593  max resid 0.03221828 
Run 16 stress 0.2495862 
Run 17 stress 0.2496288 
Run 18 stress 0.2496519 
Run 19 stress 0.2496403 
Run 20 stress 0.2496276 
*** No convergence -- monoMDS stopping criteria:
     1: no. of iterations &gt;= maxit
    19: stress ratio &gt; sratmax</code></pre>
<p><img src="figure/06_biplot.Rmd/unnamed-chunk-6-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="groot" class="section level2">
<h2>Groot</h2>
<pre class="r"><code>biplot_amr(groot_rar, metadata, RiskGroup2, 0.5)+labs(title = &quot;Groot Rarefied counts&quot;)</code></pre>
<pre><code>Run 0 stress 0.1803599 
Run 1 stress 0.1788383 
... New best solution
... Procrustes: rmse 0.03873502  max resid 0.2398207 
Run 2 stress 0.179452 
Run 3 stress 0.1824486 
Run 4 stress 0.188978 
Run 5 stress 0.18197 
Run 6 stress 0.1901835 
Run 7 stress 0.1833 
Run 8 stress 0.1776509 
... New best solution
... Procrustes: rmse 0.03414292  max resid 0.2442929 
Run 9 stress 0.1740271 
... New best solution
... Procrustes: rmse 0.02613701  max resid 0.1594867 
Run 10 stress 0.1837995 
Run 11 stress 0.1816011 
Run 12 stress 0.1746082 
Run 13 stress 0.1784326 
Run 14 stress 0.1745717 
Run 15 stress 0.1806955 
Run 16 stress 0.1771005 
Run 17 stress 0.1841789 
Run 18 stress 0.1793796 
Run 19 stress 0.1800953 
Run 20 stress 0.1875857 
*** No convergence -- monoMDS stopping criteria:
     2: no. of iterations &gt;= maxit
    18: stress ratio &gt; sratmax</code></pre>
<p><img src="figure/06_biplot.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##biplot_meta(groot_rar, metadata, RiskGroup2, 0.2)+labs(title = &quot;Groot Rarefied counts&quot;)</code></pre>
<pre class="r"><code>biplot_amr(groot_rpkm, metadata, RiskGroup2, 0.5)+labs(title = &quot;Groot RPKM&quot;)</code></pre>
<pre><code>Run 0 stress 0.1588664 
Run 1 stress 0.1642337 
Run 2 stress 0.1636576 
Run 3 stress 0.1682236 
Run 4 stress 0.156548 
... New best solution
... Procrustes: rmse 0.04547659  max resid 0.2324753 
Run 5 stress 0.1657083 
Run 6 stress 0.162819 
Run 7 stress 0.1574414 
Run 8 stress 0.1603271 
Run 9 stress 0.151538 
... New best solution
... Procrustes: rmse 0.03955989  max resid 0.2202202 
Run 10 stress 0.1557259 
Run 11 stress 0.1629994 
Run 12 stress 0.1582707 
Run 13 stress 0.1536591 
Run 14 stress 0.1497357 
... New best solution
... Procrustes: rmse 0.04236057  max resid 0.2398044 
Run 15 stress 0.1589346 
Run 16 stress 0.1555004 
Run 17 stress 0.1551458 
Run 18 stress 0.161488 
Run 19 stress 0.1491921 
... New best solution
... Procrustes: rmse 0.02279961  max resid 0.2281182 
Run 20 stress 0.1605846 
*** No convergence -- monoMDS stopping criteria:
     1: no. of iterations &gt;= maxit
    19: stress ratio &gt; sratmax</code></pre>
<p><img src="figure/06_biplot.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>biplot_meta(groot_rpkm, metadata, RiskGroup2, 0.2)+labs(title = &quot;Groot RPKM&quot;)</code></pre>
<pre><code>Run 0 stress 0.1588664 
Run 1 stress 0.1642337 
Run 2 stress 0.1636576 
Run 3 stress 0.1682236 
Run 4 stress 0.156548 
... New best solution
... Procrustes: rmse 0.04547659  max resid 0.2324753 
Run 5 stress 0.1657083 
Run 6 stress 0.162819 
Run 7 stress 0.1574414 
Run 8 stress 0.1603271 
Run 9 stress 0.151538 
... New best solution
... Procrustes: rmse 0.03955989  max resid 0.2202202 
Run 10 stress 0.1557259 
Run 11 stress 0.1629994 
Run 12 stress 0.1582707 
Run 13 stress 0.1536591 
Run 14 stress 0.1497357 
... New best solution
... Procrustes: rmse 0.04236057  max resid 0.2398044 
Run 15 stress 0.1589346 
Run 16 stress 0.1555004 
Run 17 stress 0.1551458 
Run 18 stress 0.161488 
Run 19 stress 0.1491921 
... New best solution
... Procrustes: rmse 0.02279961  max resid 0.2281182 
Run 20 stress 0.1605846 
*** No convergence -- monoMDS stopping criteria:
     1: no. of iterations &gt;= maxit
    19: stress ratio &gt; sratmax</code></pre>
<p><img src="figure/06_biplot.Rmd/unnamed-chunk-8-2.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=es_ES.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=es_ES.UTF-8        LC_COLLATE=es_ES.UTF-8    
 [5] LC_MONETARY=es_ES.UTF-8    LC_MESSAGES=es_ES.UTF-8   
 [7] LC_PAPER=es_ES.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=es_ES.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] usedist_0.4.0   glue_1.6.2      ggrepel_0.9.1   broom_1.0.0    
 [5] vegan_2.6-2     lattice_0.20-45 permute_0.9-7   forcats_0.5.1  
 [9] stringr_1.4.0   dplyr_1.0.9     purrr_0.3.4     readr_2.1.2    
[13] tidyr_1.2.0     tibble_3.1.8    ggplot2_3.3.6   tidyverse_1.3.2

loaded via a namespace (and not attached):
 [1] nlme_3.1-155        fs_1.5.2            lubridate_1.8.0    
 [4] bit64_4.0.5         httr_1.4.3          rprojroot_2.0.3    
 [7] tools_4.1.2         backports_1.4.1     bslib_0.4.0        
[10] utf8_1.2.2          R6_2.5.1            DBI_1.1.3          
[13] mgcv_1.8-39         colorspace_2.0-3    withr_2.5.0        
[16] tidyselect_1.1.2    bit_4.0.4           compiler_4.1.2     
[19] git2r_0.30.1        cli_3.3.0           rvest_1.0.2        
[22] xml2_1.3.3          labeling_0.4.2      sass_0.4.2         
[25] scales_1.2.0        digest_0.6.29       rmarkdown_2.14     
[28] pkgconfig_2.0.3     htmltools_0.5.3     highr_0.9          
[31] dbplyr_2.2.1        fastmap_1.1.0       rlang_1.0.4        
[34] readxl_1.4.0        rstudioapi_0.13     farver_2.1.1       
[37] jquerylib_0.1.4     generics_0.1.3      jsonlite_1.8.0     
[40] vroom_1.5.7         googlesheets4_1.0.0 magrittr_2.0.3     
[43] Matrix_1.4-0        Rcpp_1.0.9          munsell_0.5.0      
[46] fansi_1.0.3         lifecycle_1.0.1     stringi_1.7.8      
[49] whisker_0.4         yaml_2.3.5          MASS_7.3-55        
[52] grid_4.1.2          parallel_4.1.2      promises_1.2.0.1   
[55] crayon_1.5.1        haven_2.5.0         splines_4.1.2      
[58] hms_1.1.1           knitr_1.39          pillar_1.8.0       
[61] reprex_2.0.1        evaluate_0.15       modelr_0.1.8       
[64] vctrs_0.4.1         tzdb_0.3.0          httpuv_1.6.5       
[67] cellranger_1.1.0    gtable_0.3.0        assertthat_0.2.1   
[70] cachem_1.0.6        xfun_0.31           later_1.3.0        
[73] googledrive_2.0.0   gargle_1.2.0        workflowr_1.7.0    
[76] cluster_2.1.2       ellipsis_0.3.2     </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
