---
title: "Data summary"
author: "Elisa Rubio"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

```{r global_options, include=FALSE}
knitr::opts_chunk$set(cache=FALSE,
                      echo=TRUE, warning=FALSE, message=FALSE, results ='markup')
options(warn=-1)
```

```{r message=FALSE}
library(tidyverse)
library(kableExtra)
```

## Metadata 

Get metdatada file and check data concordance
```{r}
dataTable <- readRDS("data/Metadata/dataTable.rds")%>%select(SampleID, InitialReads)%>%
  unique()

metadata<-read_csv("data/Metadata/metadata.csv")%>%inner_join(., dataTable, by="SampleID")

ariba_samples<-list.files("data/aribaData")%>%str_extract(pattern = "Sample_\\d+")
groot_samples<-list.files("data/grootData")%>%str_extract(pattern = "Sample_\\d+")
all.equal(ariba_samples, groot_samples)
all.equal(ariba_samples, metadata$SampleID)
all.equal(groot_samples, metadata$SampleID) ##Groot samples match with metadata

metadata$SampleID[!(metadata$SampleID %in% ariba_samples)] ##missing ariba sample (Sample 83 is now included)

```

Get CARD database: 

```{r}
aro_index<-read_tsv("data/cardData/aro_index_v314.tsv")%>%
  mutate(`ARO Accession`=str_extract(`ARO Accession`, "[:digit:]+"),
         `ARO Accession`=as.numeric(`ARO Accession`))%>%                                                                   
  select(-starts_with("Model"))%>%
  unique()

str(aro_index)
```


## ARIBA

### Get all report files (list of dataframes) and assign names:

```{r}
ariba_files<-list.files("output/aribaData", full.names = TRUE, pattern ="ariba_report.tsv")
ariba_data <- lapply(ariba_files, read_tsv)
names(ariba_data)<-ariba_samples
```

### Remove contig and variant data from ARIBA:

```{r}
vars<-c("#ariba_ref_name", "ref_name" , "gene",  "var_only", "flag", "reads", "cluster", "ref_len", "free_text" )
aribasubs<-function(x){x%>%select(all_of(vars))%>%unique()}
##aribasubs2<-function(x){x%>% filter(gene==1)%>%select(c(1:8,31))%>%unique()} ##Only coding sequences
ariba_data<-lapply(ariba_data, aribasubs)
```

### Merge with CARD data in order to obtain resistance genes information (AMR gene family...)

```{r}
## test reference name structures

refname<-lapply(ariba_data, function(x) x%>%select(ref_name))
refname<-unlist(refname)%>%unique()%>%as.data.frame()%>%rename(ref_name=1)
refname<-refname%>%
  separate(ref_name, sep="\\.", into =c("ARO Term", "ARO Accession","NCBI","init_final","Model Sequence ID","v1") , remove = FALSE)%>%
  mutate(`ARO Accession` = as.numeric(`ARO Accession`),
         `ARO Accession`=ifelse(`ARO Accession` <300000, NCBI, `ARO Accession`),
        NCBI=ifelse(str_detect(init_final, "_"), NCBI, paste(NCBI, ".", init_final)),
         init_final=ifelse(str_detect(init_final, "_"), init_final, `Model Sequence ID`),
         `Model Sequence ID`=ifelse(str_detect(`Model Sequence ID`, "_"), v1, `Model Sequence ID`),
         `ARO Accession` = as.numeric(`ARO Accession`))%>%
  select(-v1, -`Model Sequence ID`)

refname_all_ariba<-left_join(refname, aro_index, by="ARO Accession")

head(refname_all_ariba, 8)%>%kable(caption="**Ariba detected ARG with CARD information** ")%>%kable_paper("striped")%>%scroll_box(width = "100%")
```


```{r}
aribanotation<-function(x, aro_index){
  x<-x%>%separate(ref_name, sep="\\.", into =c("ARO Term", "ARO Accession","NCBI","init_final","Model Sequence ID","v1") , remove = FALSE)%>%
  mutate(`ARO Accession` = as.numeric(`ARO Accession`),
         `ARO Accession`=ifelse(`ARO Accession` <300000, NCBI, `ARO Accession`),
        NCBI=ifelse(str_detect(init_final, "_"), NCBI, paste(NCBI, ".", init_final)),
         init_final=ifelse(str_detect(init_final, "_"), init_final, `Model Sequence ID`),
         `Model Sequence ID`=ifelse(str_detect(`Model Sequence ID`, "_"), v1, `Model Sequence ID`),
        `ARO Accession` = as.numeric(`ARO Accession`))%>%
  select(-v1, -`Model Sequence ID`)%>%inner_join(aro_index, by="ARO Accession")
  return(x)}

ariba_data<-lapply(ariba_data, function (x) aribanotation (x, aro_index ))

```

### Calculate RPKM 

Old way to calculate (taking into account all reads per sample)
```{r eval=FALSE, include=FALSE}
metadata_ariba<-metadata%>%filter(SampleID %in% ariba_samples)

for (i in 1:length(ariba_data)){
 ariba_data[[i]]$RPKM<-ariba_data[[i]]$`reads`/(ariba_data[[i]]$`ref_len`/1000 * metadata_ariba$InitialReads[i]/1000000)}
```

RPKM: reads per million mapped reads (we take into account mapped reads instead of all reads)
```{r eval=TRUE, include=TRUE}
ariba_mapped_reads<-lapply(ariba_data, function(x) sum(x$reads))%>%unlist()%>%as.data.frame()%>%
  rownames_to_column("SampleID")%>%tibble()%>%rename(mapped_reads=2)

all.equal(ariba_mapped_reads$SampleID, ariba_samples)

for (i in 1:length(ariba_data)){
 ariba_data[[i]]$RPKM<-ariba_data[[i]]$`reads`/(ariba_data[[i]]$`ref_len`/1000 * ariba_mapped_reads$mapped_reads[i]/1000000)}
```


### Transform data to obtain matrix 

```{r}
transform_data<-function(list_df, sample_names, df_variable){
  df_variable<-enquo(df_variable)
df_filt<-lapply(list_df, function(x) select(x, ref_name, !!df_variable)) ##select variables

for (i in 1:length(df_filt)){
 names(df_filt[[i]])[2]<-sample_names[i]} ##change reads column name to sample name

df_transform<-df_filt %>% reduce(full_join, by='ref_name')
df_transform[is.na(df_transform)] <- 0
return(df_transform)}
```

```{r}
ariba_rpkm<-transform_data(ariba_data, ariba_samples, RPKM)
head(ariba_rpkm, 15)%>%kable(caption="**Table 1. Ariba RPKM** ")%>%kable_paper("striped")%>%scroll_box(width = "100%")
```

```{r}
ariba_counts<-transform_data(ariba_data, ariba_samples, reads)
head(ariba_counts, 15)%>%kable(caption="**Table 1. Ariba Counts** ")%>%kable_paper("striped")%>%scroll_box(width = "100%")
```


## Groot

### Get all report files (list of dataframes) and assign names:

```{r}
groot_files<-list.files("output/grootData", full.names = TRUE, pattern ="groot.report.tsv")
groot_data <- lapply(groot_files, function(x) read_tsv (x, col_names = c("ref_name", "read_count", "gene_length", "coverage_cigar")))
names(groot_data)<-groot_samples
```

### Merge with CARD data in order to obtain resistance genes information (AMR gene family...)

```{r}
## test reference name structures

refname_groot<-lapply(groot_data, function(x) x%>%select(ref_name))%>%unlist()%>%unique()%>%as.data.frame()%>%rename(ref_name=1)

refname_groot<-refname_groot%>%
  separate(ref_name, sep="\\.", into =c("ARO Term", "ARO Accession","NCBI","init_final","Model Sequence ID","v1") , remove = FALSE)%>%
  mutate(`ARO Accession` = as.numeric(`ARO Accession`),
         `ARO Accession`=ifelse(`ARO Accession` <300000, NCBI, `ARO Accession`),
        NCBI=ifelse(str_detect(init_final, "-"), NCBI, paste(NCBI, ".", init_final)),
         init_final=ifelse(str_detect(init_final, "-"), init_final, `Model Sequence ID`),
         `Model Sequence ID`=ifelse(str_detect(`Model Sequence ID`, "-"), v1, `Model Sequence ID`),
         `ARO Accession` = as.numeric(`ARO Accession`))%>%
  select(-v1)
refname_all_groot<-left_join(refname_groot, aro_index, by="ARO Accession")

which(is.na(refname_all_groot$`AMR Gene Family`))
refname_all_groot%>%slice(n=569) ##one missig arg gene, not in card database.

##Searched for the ARO accession and the NCBI accession and there was nothing as a resistance gene to chloramphenicol. 

head(refname_all_groot, 8)%>%kable(caption="**Groot detected ARG with CARD information** ")%>%kable_paper("striped")%>%scroll_box(width = "100%")
```

```{r}
grootnotation<-function(x, aro_index){
  
  x<-x%>%separate(ref_name, sep="\\.", into =c("ARO Term", "ARO Accession","NCBI","init_final","Model Sequence ID","v1") , remove = FALSE)%>%
  mutate(`ARO Accession` = as.numeric(`ARO Accession`),
         `ARO Accession`=ifelse(`ARO Accession` <300000, NCBI, `ARO Accession`),
        NCBI=ifelse(str_detect(init_final, "-"), NCBI, paste(NCBI, ".", init_final)),
         init_final=ifelse(str_detect(init_final, "-"), init_final, `Model Sequence ID`),
         `Model Sequence ID`=ifelse(str_detect(`Model Sequence ID`, "-"), v1, `Model Sequence ID`),
         `ARO Accession` = as.numeric(`ARO Accession`))%>%
  select(-v1)%>%left_join(aro_index, by="ARO Accession")%>%unique()
  return(x)}
  
groot_data<-lapply(groot_data, function (x) grootnotation (x, aro_index ))
```

### Calculate RPKM

```{r eval=FALSE, include=FALSE}
for (i in 1:length(groot_data)){
 groot_data[[i]]$RPKM<-groot_data[[i]]$read_count/(groot_data[[i]]$gene_length/1000 * metadata$InitialReads[i]/1000000)
}
```

RPKM: reads per million mapped reads (we take into account mapped reads instead of all reads)
```{r eval=TRUE, include=TRUE}
groot_mapped_reads<-lapply(groot_data, function(x) sum(x$read_count))%>%unlist()%>%as.data.frame()%>%
  rownames_to_column("SampleID")%>%tibble()%>%rename(mapped_reads=2)

all.equal(groot_mapped_reads$SampleID, groot_samples)

for (i in 1:length(groot_data)){
 groot_data[[i]]$RPKM<-groot_data[[i]]$read_count/(groot_data[[i]]$gene_length/1000 * groot_mapped_reads$mapped_reads[i]/1000000)
}
```

### Transform data to obtain matrix 

```{r}
groot_rpkm<-transform_data(groot_data, groot_samples, RPKM)
head(groot_rpkm, 15)%>%kable(caption="**Table. Groot RPKM** ")%>%kable_paper("striped")%>%scroll_box(width = "100%")
```

```{r}
groot_counts<-transform_data(groot_data, groot_samples, read_count)
head(groot_counts, 15)%>%kable(caption="**Table. Groot Counts** ")%>%kable_paper("striped")%>%scroll_box(width = "100%")
```

## Save obtained matrix and AMR genes metadata

```{r}
save(groot_rpkm, groot_counts, refname_all_groot, file="output/summarized_groot.RDA")
save(ariba_rpkm, ariba_counts, refname_all_ariba, file="output/summarized_ariba.RDA")
```

## Save AMR genes information in csv file:

```{r}
write_excel_csv(refname_all_ariba, "output/refname_all_ariba.csv")
write_excel_csv(refname_all_groot, "output/refname_all_groot.csv")
```

